helm-chart/amalthea-sessions/templates/amaltheasession-crd.yaml

```diff
@@ -4011,6 +4011,12 @@ spec:
                       rule: self == oldSelf
                   ingressClassName:
                     type: string
+                  pathPrefix:
```

Comment:
Shouldn't these modifications be done in `bundle/manifests/amalthea.dev_amaltheasessions.yaml` as well?